src/common/entities/User.entity.ts

```diff
@@ -0,0 +1,19 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+@Entity()
+export class User {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  email: string;
+
+  @Column()
+  password: string;
```

Comment:
does it required?

Reply:
if i understood righly it's not required because of Google Auth case

Reply:
@lyudad fixed